src/map_field.cpp

```diff
@@ -476,6 +476,7 @@ bool map::process_fields_in_submap( submap *const current_submap,
                     sblk.apply_slime( p, cur.get_field_intensity() * curtype.obj().apply_slime_factor );
                 }
                 if( curtype == fd_fire ) {
+                    cur.set_field_age( std::min( 24_hours, cur.get_field_age() ) );
```

Comment:
Kinda dumb thing about fire "age", it's negative.
If a fire is "10 minutes old" it's at great risk of going out, if it has a negative age it's in no risk of going out.
So this needs to cap at -24_hours

Reply:
hmm, til.